AUTHORS.rst

```diff
@@ -63,4 +63,15 @@ The following people have been core contributors to scikit-learn's development a
   * `Jake VanderPlas <http://www.astro.washington.edu/users/vanderplas/>`_
   * Nelle Varoquaux
   * `Gael Varoquaux <http://gael-varoquaux.info/blog/>`_
-  * Ron Weiss
+  * Ron Weissz
+
+Please do not email the authors directly with bug reports or feature requests.
+To address those concerns, please make use of the
+`issue tracker on Github <https://github.com/scikit-learn/scikit-learn/issues>`_
+or the `mailing list <https://mail.python.org/mailman/listinfo/scikit-learn>`_.
+If you are not sure whether you have a found a bug,
+or you have a more general question about
```

Comment:
this phrase sounds a bit odd...shouldn't it be "if you are not sure whether you have found a bug, or if you have a more general question". could be just me.